Updated some translation files

cluster28 opened the pull request:
Basque translations

| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #
| License       | MIT
| Doc PR        | symfony/symfony-docs

<!--
Write a short README entry for your feature/bugfix here (replace this comment block.)
This will help people understand your PR and can be used as a start of the Doc PR.
Additionally:
 - Bug fixes must be submitted against the lowest branch where they apply
   (lowest branches are regularly merged to upper ones so they get the fixes too).
 - Features and deprecations must be submitted against the master branch.
-->

cluster28 committed:
Updated some translation files

stof reviewed (approved):
I will assume that the translations themselves are good. I don't think we have anyone in the core team speaking Basque to review them (I may be wrong though)

stof commented:
Can you also provide the few missing translations in https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Resources/translations/validators.eu.xlf (English goes to id 93 while Basque only has them until id 85) ?

cluster28 commented:
> I will assume that the translations themselves are good. I don't think we have anyone in the core team speaking Basque to review them (I may be wrong though)

Maybe @javiereguiluz? He's from Vitoria.

cluster28 commented:
> Can you also provide the few missing translations in https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Resources/translations/validators.eu.xlf (English goes to id 93 while Basque only has them until id 85) ?

I'm going to see what can I do...

javiereguiluz commented:
Yes, I understand some Basque. Translation looks correct to me, but we could ping our friend @ikerib who speaks Basque perfectly and contributed some translations in the past! Thanks.

javiereguiluz commented on src/Symfony/Component/Validator/Resources/translations/validators.es.xlf:
```diff
@@ -330,10 +330,38 @@
             </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Esta contraseña ha sido desvelada por un fallo de seguridad y no debería volver a ser utilizada. Por favor utilice otra contraseña</target>
```
This message is very difficult to translate. I'd recommend an alternative translation to clearly explain that this is not a security problem of our own application:

```
Esta contraseña no se puede utilizar porque está incluida en un listado de contraseñas públicas obtenido gracias a fallos de seguridad de otros sitios y aplicaciones. Por favor utilice otra contraseña.
```

cluster28 replied:
May french translation should be rewieved beacause I've translated it from [this line](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf#L363)

Simperfit replied:
The french translation is right btw.

javiereguiluz replied:
@Simperfit I'm not sure about that. First, "data breach" should be translated as [violation de données](https://fr.wikipedia.org/wiki/Violation_de_donn%C3%A9es) instead of "faille de sécurité". Second, the way it's written could mean that the security problem is caused by the web site or application displaying the message.

Simperfit replied:
I understand what you mean, for me it does not mean that the current application showing the message is one who has been hacked / selled.
But It can be improved with what you proposed to have the same meaning in french. 

smth like: "Ce mot de passe ne peut pas être utilisé car il fait partie d'une liste de mots de passe publics obtenus grâce aux fuites de données liée à d'autres sites et applications. Veuillez utiliser un autre mot de passe."
WDYT ?

javiereguiluz replied:
It's super long ... but it describes the problem with absolute precision and I think most users can understand it. I wish we could find a more concise message while still explaining things well, but I think we can't.

ikerib commented:
> Yes, I understand some Basque. Translation looks correct to me, but we could ping our friend @ikerib who speaks Basque perfectly and contributed some translations in the past! Thanks.

This was done before
https://github.com/symfony/symfony/pull/30223/commits/f496ab638e0f9161812938e942f5fb22012b20a3

Do I need to do it again?

javiereguiluz commented:
@ikerib in f496ab6 you contributed the translations related to "validators" ... but in Symfony we have other translation files. In this PR, @cluster28 is translating the "security" file. So, we wonder if you have some time to double check these new translations and propose some fixes if needed. Thanks!

ikerib commented:
Ahhhh I see. Ok, I’m on holidays now but I’ll do it as soon as possible ;-)

nicolas-grekas commented:
Why does this PR update `security.eu.xlf` and `validators.es.xlf` but not `validators.eu.xlf`?

javiereguiluz commented:
@nicolas-grekas the title of the PR is misleading. We're updating two different languages in the same PR.

nicolas-grekas commented:
Should we merge as is for now?

Stadly committed:
[Translator] Preserve default domain when extracting strings from php files

fabpot committed:
bug #31275 [Translator] Preserve default domain when extracting strings from php files (Stadly)

This PR was squashed before being merged into the 3.4 branch (closes #31275).

Discussion
----------

[Translator] Preserve default domain when extracting strings from php files

| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

Use default domain when extracting translation strings with `null` as domain.

Commits
-------

ca77d9a33e [Translator] Preserve default domain when extracting strings from php files

fabpot commented:
Thank you @cluster28.
